src/bin/cql-stress-cassandra-stress/java_generate/values/hex_blob.rs

```diff
@@ -0,0 +1,191 @@
+use super::ValueGenerator;
+use crate::java_generate::distribution::Distribution;
+
+/// Based on c-s HexBytes generator.
+/// Used by c-s to generate blob partition keys.
+/// TBH, not quite sure why c-s won't use Bytes generator for this.
+pub struct HexBlob;
+
+impl ValueGenerator for HexBlob {
+    type Value = Vec<u8>;
+
+    fn generate(
+        &mut self,
+        identity_distribution: &mut dyn Distribution,
+        size_distribution: &mut dyn Distribution,
+    ) -> Self::Value {
+        let seed = identity_distribution.next_i64();
+        size_distribution.set_seed(seed);
+        let size = size_distribution.next_i64() as usize;
+
+        let mut result = vec![0; size];
```

Comment:
Why not `let mut result = Vec::with_capacity(size);` and then `result.push()` the bytes? This will skip the zero initialization of the vector.